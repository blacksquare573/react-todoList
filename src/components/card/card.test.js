import Card from "./card.component";
import renderer from "react-test-renderer";

test("renders correctly", () => {
  const mockMonster = {
    name: "jack",
    id: 1,
    email: "qq1111@gmail.com",
  };

  const tree = renderer.create(<Card monster={mockMonster} />).toJSON();
  expect(tree).toMatchSnapshot();
});
