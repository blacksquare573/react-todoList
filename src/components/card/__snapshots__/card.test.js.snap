// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="card-container"
>
  <img
    alt="monster jack"
    src="https://robohash.org/1?set=set2&size=180x180"
  />
  <h2>
    jack
  </h2>
  <p>
    qq1111@gmail.com
  </p>
</div>
`;
